src/Common/SegmentedBarChart/types.ts

```diff
@@ -18,11 +18,13 @@ export type Entity = {
     color: string
     label: string
     value: number
+    proportionalValue?: string
```

Comment:
This prop is redundant since I am calculating total based on the value. Can discuss on this.